x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -202,8 +204,33 @@ function useServicesDetailedStatisticsFetcher({
 }
 
 export function ServiceInventory() {
+  const {
+    core: { analytics },
+  } = useApmPluginContext();
+
+  const fetchMainStatisticsStartTime = useRef(0);
```

Comment:
Can we put this in a separate hook? Or maybe as an option on useFetcher? I think @sqren might have ideas here.

Reply:
i actually started thinking in that direction that may be we can expose 2 variables with the useFetcher
1. `shouldReportMetrics` which will be default false, and can be enabled.
2. A custom event name, which can be passed to if the above is set to true.

But then that would have required a bit more effort in refactoring a lot of code. So open for idea here

Reply:
We discussed this offline and agreed to move the time measuring to the journey itself if possible. 
If that's not possible I suggested to calculate latency for APIs in x-pack/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts